docs/plugins.rst

```diff
@@ -166,6 +166,9 @@ plugin under consideration.  The objects themselves are relatively simple:
       enabled: false
       path: path/to/quux
 
+    - name: docs
+      enabled: true
+
```

Comment:
It appears this has changed to an associative array, so instead of

```
  - name: docs
    enabled: true
```

you would have

```
  docs:
    enabled: true
```

Am I correct that this documentation should be changed?